justfile

```diff
@@ -1,58 +1,77 @@
 set dotenv-load := false
 
+# Show all available recipes
+default:
+  @just --list --unsorted
```

Comment:
TIL! This is perfect, I use `just -lu` all the time, I had no idea it could be set as the default :100: